### ISSUE TITLE
iMac Fusion Drive failed to load checkpoint map

### ISSUE THREAD

**Issue by drichte1** [sgan81/apfs-fuse#108]
I want to mount a Fusion Drive on an iMac with MacOS 10.14. I am running a Ubuntu 19.10 from USB flash drive and compiled fuse-apfs from the latest git repo.
It is an iMac late 2015 so there is no TPM chip!! 

The disks are not broken and I don't use encryption, I just want to try it but I always get this error messages:

sudo apfs-fuse -d 255 -l -o allow_other -f /dev/sdb /dev/sda /mnt
Device /dev/sda opened. Size is 2000398934016
Device /dev/sdb opened. Size is 121332826112
Found valid GPT partition table. Looking for APFS partition.
Found valid GPT partition table on secondary device. Looking for APFS partition.
Failed to load checkpoint map
Unable to load container.

Any idea?

**Comment by sgan81** [comment]
Yes, swap the drives ... -f /dev/sda /dev/sdb. Looks like sdb is the primary drive and sda the secondary one.
